mmaction/models/recognizers/base.py

```diff
@@ -125,40 +125,45 @@ def extract_feat(self, imgs):
             x = self.backbone(imgs)
         return x
 
-    def average_clip(self, cls_score, num_segs=1):
```

Comment:
Re-implement `average_clip` using `aggregate_clip` instead of removing it. And add a warning to tell user to use `aggregate_clip`. Also give the link of this pr for reference.